config-overrides.js

```diff
@@ -0,0 +1,21 @@
+// config-overrides.js
+const StylelintPlugin = require('stylelint-webpack-plugin')
+
+module.exports = {
+  webpack: function (config, env) {
+    if (env === 'development') {
+      config.plugins.push(
+        new StylelintPlugin({
+          configFile: 'stylelint.config.js',
+          context: 'src',
+          files: '**/*.scss',
```

Comment:
Does this get all files? ie how about more nested `.scss` files - i think you can pass this an Array of globs

Reply:
You can pass an array of globs string yes but it already checks nested `.scss` files.

In the style lint configs, I set implicitComponents to true which will check all files within context ('src'). 

<img width="616" alt="Screen Shot 2020-08-05 at 3 05 50 PM" src="https://user-images.githubusercontent.com/18355766/89468930-53580900-d72d-11ea-9fe8-b63709366935.png">